ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -97,31 +121,45 @@ export class InstallationProgress extends React.Component {
         }
 
         return (
-            <Bullseye className={idPrefix + "-status-" + status}>
-                <EmptyStatePanel
-                  icon={icon}
-                  loading={!icon}
-                  paragraph={
-                      status !== "success"
-                          ? <Progress
-                              id={idPrefix + "-bar"}
-                              label={cockpit.format("$0 of $1", step, steps)}
-                              max={steps}
-                              min={0}
-                              title={message}
-                              value={step}
-                              valueText={cockpit.format("$0 of $1", step, steps)}
-                              variant={status}
-                          />
-                          : <Text>{_("Fedora is successfully installed. To begin using it, reboot your system.")}</Text>
-                  }
-                  secondary={
-                      status === "success" &&
-                      <Button onClick={exitGui}>{_("Reboot")}</Button>
-                  }
-                  title={title}
-                />
-            </Bullseye>
+            <EmptyStatePanel
+              icon={icon}
+              loading={!icon}
+              paragraph={
+                  <Flex direction={{ default: "column" }}>
+                      <ProgressStepper isCenterAligned>
+                          {progressSteps.map((progressStep, index) => {
+                              let variant = "pending";
+                              const ariaLabel = _("pending step");
+                              if (index < currentProgressStep) {
+                                  variant = "success";
+                              } else if (index === currentProgressStep) {
+                                  variant = "info";
+                              }
+                              return (
+                                  <ProgressStep
+                                    aria-label={ariaLabel}
+                                    id={idPrefix + "-progress-step-" + index}
+                                    isCurrent={index === currentProgressStep}
+                                    titleId={progressStep.id}
+                                    key={index}
+                                    variant={variant}
+                                  >
+                                      {progressStep.title}
+                                  </ProgressStep>
+                              );
+                          })}
+                      </ProgressStepper>
+                      <Text component={TextVariants.p}>
```

Comment:
I am almost sure, that for the styles for the Text to be applied, you need to wrap it in a TextContent.

Reply:
Lets do that. :)